src/Tools/CoreFx.Tools/GenerateConfigurationProps.cs

```diff
@@ -111,6 +119,32 @@ private void ParseConfigurationString(string propertyName, ProjectRootElement pr
             }
         }
 
+        private void CreateRuntimeIdentifier(ProjectRootElement project)
+        {
+            var rid = Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment.GetRuntimeIdentifier();
+
+            string[] ridParts = rid.Split('-');
+
+            if (ridParts.Length < 1)
+                throw new System.InvalidOperationException($"Unknwon rid format {rid}.");
+
+            string osNameAndVersion = ridParts[0];
+
+            // For now force the rid to always be win7 for windows.
+            if (osNameAndVersion.StartsWith("win"))
+            {
+                osNameAndVersion = "win7";
```

Comment:
Yeah I only kept it this way because it was defaulting to that before my changes (to detect the default test RID), and I didn't want to cause too much churn. For now, we should just remove any special-casing there and see how it goes.